app/models/dialog_field_sorted_item.rb

```diff
@@ -93,7 +93,7 @@ def sort_data(data_to_sort)
   def raw_values
     @raw_values ||= dynamic ? values_from_automate : self[:values].to_miq_a
     unless @raw_values.collect { |value_pair| value_pair[0] }.include?(default_value)
-      self.default_value = sort_data(@raw_values).first.first
+      self.default_value = sort_data(@raw_values).first.try(:first)
```

Comment:
@gmcculloug Yes, that is correct.

Maybe this has to do with timing, but the other alternative would be to check for an empty array and in that case not set the `default_value`. On line 98, though, we're using `default_value`, and it's already been established by line 95 that the `default_value` is non-existent within the `@raw_values` array, so setting it to nil is probably the better option since you shouldn't be able to select a value that isn't there as opposed to just not selecting a value.